### PR TITLE
feat(webapp): improve homepage

### LINKED ISSUE #198: Improve Homepage
We are going to design a home page so that the new user feels attracted to investigate and use our page.

Image

### PR DESCRIPTION
**Pull request by warnerHurtado**
### What does this PR do?

- Resolve #198 

### Steps to test

1. run the project and open localhost:3000 in your browser
2. click on home in the sidebar and see the new features

#### CheckList

- [X] Follow proper Markdown format
- [X] The content is adequate
- [X] The content is available in both english and spanish
- [X] I Ran a spell check

### CODE REVIEW

**Review by JAMares — APPROVED** [review]
LGTM

File: webapp/src/routes/Home/index.js [hunk 1]
```diff
@@ -1,15 +1,81 @@
...
   const { t } = useTranslation('homeRoute')
 
   return (
-    <Box>
-      <Typography>{t('welcomeMessage')}</Typography>
-    </Box>
+    <div>
+      <div className={classes.titleContainer}>
+        <div>
+          <span className={classes.titleStyles}>Eden Expend Explorer</span>
```
Comment (xavier506) [inline webapp/src/routes/Home/index.js:18]: Eden **Spend** Explorer

**Review by xavier506 — CHANGES_REQUESTED** [review]
See comment

**Review by xavier506 — APPROVED** [review]
LGTM

File: webapp/src/routes/Home/index.js [hunk 2]
```diff
@@ -1,15 +1,90 @@
...
-import Typography from '@mui/material/Typography'
+
+import styles from './styles'
+
+const useStyles = makeStyles(styles)
 
 const Home = () => {
+  const classes = useStyles()
+  const navigate = useNavigate()
+
```
Comment (tetogomez) [inline webapp/src/routes/Home/index.js:14]: remove this black space

File: webapp/src/routes/Home/index.js [hunk 3]
```diff
@@ -1,15 +1,90 @@
...
+    })
+  }
+
   return (
-    <Box>
-      <Typography>{t('welcomeMessage')}</Typography>
-    </Box>
+    <div>
+      <div className={classes.titleContainer}>
+        <div>
```
Comment (tetogomez) [inline webapp/src/routes/Home/index.js:26]: is this `div` necessary?

File: webapp/src/routes/Home/styles.js [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+    marginTop: '16px',
+    '& .css-1exfphm-MuiButtonBase-root-MuiButton-root': {
+      width: '191px',
+      height: '36px',
+      backgroundColor: '#2563eb',
+      borderRadius: 0,
+      padding: '6px 12px',
+      textTransform: 'none'
+    },
+    '& .css-1exfphm-MuiButtonBase-root-MuiButton-root:hover': {
```
Comment (tetogomez) [inline webapp/src/routes/Home/styles.js:14]: long classes wrong

**Review by tetogomez — CHANGES_REQUESTED** [review]
check the comments

**Pull request merged**